packages/governance/test/unitTests/test-committee.js

```diff
@@ -134,7 +134,28 @@ test('committee-open question:one', async t => {
   );
 });
 
-test('committee-open question:mixed', async t => {
+const snapshotStorage = async (t, storage) => {
+  // chainStorage publication is unsynchronized
+  await eventLoopIteration();
+
+  const live = [...storage.keys()].map(key => [
+    key.replace(
+      'mockChainStorageRoot.thisElectorate.',
+      'published.committees.Economic_Committee.',
+    ),
+    storage.getBody(key),
+  ]);
+
+  const note = `Under "published", the "committees.Economic_Committee" node is delegated to
+a committee contract, which publishes data as follows.
```

Comment:
It's supposed to be illustrative. Ideally, it would illustrate all branches of any union types. I'm not sure how to be precise about what the reader can count without putting so much text in the way that it's distracting..

> A plausible instance?

That's accurate and concise, if not complete. I suppose I can go with that.